core/src/client/test_client.rs

```diff
@@ -542,10 +542,17 @@ impl BlockChainClient for TestBlockChainClient {
         self.miner.ready_transactions(range)
     }
 
+    fn future_ready_transactions(&self, range: Range<u64>) -> PendingSignedTransactions {
+        self.miner.future_ready_transactions(range)
```

Comment:
I'm curious if there is a better name.

I think, in this context, "ready" means the "current". For me, it's hard to make guess the behavior from this name.